code/game/machinery/ARES/ARES_interface.dm

```diff
@@ -315,6 +326,9 @@
 		if("page_deleted_1to1")
 			last_menu = current_menu
 			current_menu = "deleted_talks"
+		if("page_tech")
+			last_menu = current_menu
+			current_menu = "tech_log"
 
 		// -- Delete Button -- //
 		if("delete_record")
```

Comment:
You have not implemented deleting a record in the `switch(record.record_name)` but currently have the delete button allowed:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/8624f7de-e98c-47f3-b9fe-7e487c1b9962)

Clicking and confirming a deletion does nothing for the user for this kind of record currently.